project/iar1000/README.md

```diff
@@ -0,0 +1,98 @@
+# Voronoiagram
+
+## Problem to solve
+
+The Voronoi Diagram $D$ describes a partitioning of a plane $\Omega$ into a set of cells $C$, given a set of target points $T$.
+Each cell $c_k \in C$ defines the set of points consisting of one target point $t_k \in T$ and all points in $\Omega$ that are closer to $t_k$ than to any other point in $T$, i.e.
+
+$$
+c_k = \{ x \in \Omega | d(x, t_k) \leq d(x,t_q), \forall t_q\in T. t_q \neq t_k\}.
+
+$$
+
+, whereas the metric "closer" is defined by some distance function $d: \Omega \times \Omega \rightarrow \mathbb{R}$.
+
+### Fortune's Algorithm
+
+The algorithm uses a __sweep line__ and a __beach line__ to create and grow new cells of the diagram. The __sweep line__ is a vertical line "moving" in one direction (we assume the direction to be from left to right from now on) through the plane, whereas points passed by the sweep line start growing their cells. The __beach line__ is a piecewise parabolar curve, consisting of a set of Arcs P, tracking cell boarders that are still growing. The beach line describes all points that are equidistant from the sweep line to the nearest target point.
+
+![](https://upload.wikimedia.org/wikipedia/commons/0/0c/Fortunes-algorithm-slowed.gif)
+
+## System Architecture
```

Comment:
The contents you have in the section of system architecture, except the interface, serve more like the mathematical or numerical architecture.  It is the foundation for how the software will be architected and designed, but not exactly the software architecture.

It is well written and should help your development.  It is a good way to start a numerical project.  I hope pointing out the difference to the software architecture will help you reach a more versatile system down the road.